core/templates/dev/head/pages/exploration_player/exploration_player.html

```diff
@@ -87,22 +87,26 @@ <h1 class="oppia-exploration-h1"><span class="protractor-test-exploration-header
   <script src="/extensions/objects/templates/SanitizedUrlEditor.js"></script>
   <script src="/extensions/objects/templates/ListOfTabsEditor.js"></script>
 
-  <script src="/templates/dev/head/services/AutoplayedVideosService.js"></script>
-  <script src="/templates/dev/head/services/ExplorationHtmlFormatterService.js"></script>
-  <script src="/templates/dev/head/services/ExtensionTagAssemblerService.js"></script>
-  <script src="/templates/dev/head/services/MessengerService.js"></script>
+  <script src="/templates/dev/head/services/AssetsBackendApiService.js"></script>
```

Comment:
Thanks for sorting the imports here and elsewhere! Looks better now :)

Reply:
It's something I always end up doing in my PRs :P